winrm/client.go

```diff
@@ -2,19 +2,25 @@ package winrm
 
 import (
 	"bytes"
-	"crypto/tls"
 	"crypto/x509"
 	"fmt"
 	"io"
-	"net/http"
+	"launchpad.net/gwacl/fork/http"
```

Comment:
You need to use that fork, otherwise you will get an error:
```unknown error Post https://192.168.5.7:5986/wsman: local error: no renegotiation```

Also, check this https://github.com/golang/go/issues/5742